frontend/src/modules/report/report-charts.js

```diff
@@ -13,6 +15,30 @@ const defaultChartOptions = {
     '#06B6D4',
     '#F97316'
   ],
+  library: {
```

Comment:
By changing this here, some chart types are overriding it later on the file. For example donut/pie graph has a specific configuration for plugins.
Maybe add this configuration to the bottom of the file where we are returning
```
return {
    ...defaultChartOptions,
    ...chartTypeOptions
  }
```

If you check the current default report in the "Activities by Type" donut chart, you can still see the old labels

Reply:
Updated this part :+1:  When there's a dimension, we'll still use old style tooltips (like dimension: value). Graphs without dimensions will use the new style tooltips